spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -34,8 +88,65 @@ When running in an environment where Jakarta Transactions and Jakarta CDI are av
 
 When a repository method is annotated with an interceptor binding annotation, the interceptor is bound to the repository bean according to the interceptor binding annotation of the repository interface method, causing the bound interceptor to be invoked around the repository method when it runs. This enables the use of interceptors such as `jakarta.transaction.Transactional` on repository methods when running in an environment where the Jakarta EE technology that provides the interceptor is available.
 
+
 === Jakarta Persistence
 
+When integrating Jakarta Data with Jakarta Persistence, developers can leverage the JPA annotations to define the mapping of entities in repositories. Entities in Jakarta Persistence are typically annotated with `jakarta.persistence.Entity` to indicate their persistence capability.
+
+A Jakarta Data provider that supports Jakarta Persistence allows you to define repositories for classes marked with the `jakarta.persistence.Entity` annotation.
+
+By supporting Jakarta Persistence annotations, Jakarta Data providers enable Java developers to utilize familiar and standardized mapping techniques when defining entities in repositories, ensuring compatibility and interoperability with the respective technologies.
+
 === Jakarta NoSQL
 
-=== Jakarta Validation
\ No newline at end of file
+When integrating Jakarta Data with Jakarta NoSQL, developers can use the NoSQL annotations to define the mapping of entities in repositories. Entities in Jakarta NoSQL are typically annotated with `jakarta.nosql.Entity` to indicate their suitability for persistence in NoSQL databases.
+
+A Jakarta Data provider that supports Jakarta NoSQL will scan classes marked with the `jakarta.nosql.Entity` annotation.
+
+By supporting Jakarta NoSQL annotations, Jakarta Data providers enable Java developers to utilize familiar and standardized mapping techniques when defining entities in repositories, ensuring compatibility and interoperability with the respective technologies.
+
+=== Jakarta Bean Validation
+
+Integrating with Jakarta Validation ensures data consistency within the Java layer. By applying validation rules to the data, developers can enforce constraints and business rules, preventing invalid or inconsistent information from being processed or persisted.
+
+Incorporating Jakarta Validation into your Jakarta Data applications helps enforce data consistency at the Java layer. Whenever data is converted to or from Java objects, the validation rules defined in the entity are automatically executed. For instance, when persisting a `Student` object, the `name` field will be validated to ensure it is not blank.
```

Comment:
We need to replace the text, "Whenever data is converted to or from Java objects" because it would require that validation be applied on find operations. Here is what we can change it to:
```suggestion
Incorporating Jakarta Validation into your Jakarta Data applications helps enforce data consistency at the Java layer. When you invoke the `save` or `saveAll` method on repositories, the Jakarta Data provider ensures that the validation rules defined in the entity are automatically executed prior to updating the database if a Jakarta Validation provider is present. For instance, when persisting a `Student` object, the `name` field will be validated to ensure it is not blank.
```